bin/livy-server

```diff
@@ -32,9 +54,115 @@ if [ ! -d "$LIBDIR" ]; then
   exit 1
 fi
 
-CLASSPATH="$LIBDIR/*:$LIVY_HOME/conf:$CLASSPATH"
 
-exec java \
-	$LIVY_SERVER_JAVA_OPTS \
-	-cp "$CLASSPATH" \
-	com.cloudera.livy.server.Main
+
+
+if [ "$LIVY_IDENT_STRING" = "" ]; then
+  export LIVY_IDENT_STRING="$USER"
+fi
+
+
+# get log directory
+if [ "$LIVY_LOG_DIR" = "" ]; then
+  export LIVY_LOG_DIR="${LIVY_HOME}/logs"
+fi
+mkdir -p "$LIVY_LOG_DIR"
+touch "$LIVY_LOG_DIR"/.livy_test > /dev/null 2>&1
```

Comment:
`set -e` is set somewhere above this line, so if this fails, nothing else will execute.